src/mobile/FooterLinks/FooterLinks.scss

```diff
@@ -0,0 +1,53 @@
+@import "../../scss/variables.scss";
+
+.footerLinks {
+  display: flex;
+  flex-wrap: wrap;
+
+  .socialMedia {
+    max-width: 370px;
+    width: 100%;
+    display: flex;
+    gap:20px;
+    padding-bottom: 20px;
+
+    .socialMediaItem {
+      width: 47px;
```

Comment:
Think about moving such data to constants in `variables.scss`. You could divide this file with constants by topics.